src/main/resources/config-env/artemis-dev-db.env

```diff
@@ -0,0 +1 @@
+SPRING_PROFILES_ACTIVE=dev,bamboo,bitbucket,jira,artemis,scheduling,athene
```

Comment:
I do not quite understand the naming of this file: Why is it called `-db` (which I would understand as `database`) but it seems to contain the atlassian stack (as opposed to `artemis-dev-jenkins.env`)?

Reply:
My idea was to base the `.env` files on the docker-compose setups and the spring profiles.

So I used `artemis-<profile>.env` or just `<profile.env> (maybe it would be good to make this consistent :hushed: ) for the spring profiles, for instance, `artemis-dev.env` for the profile `application-dev.yml` and `artemis.env` just for the default `application.yml`.

The files like `artemis-dev-db.yml` (belongs to the docker-compose setup with just artemis and a db) are specific to the docker compose setup and contain all the default environment of these compose files not having to mix `environment` and `env_file` keys in the compose settings. This is also where the `artemis-override.env` comes into play which is an override for your not versioned envs.

All of this also still has to be documented if this solution makes sense :wink: as this was just an idea to make it somehow working for multiple docker-compose settings.

Also, I would still like to discuss if this approach makes sense as I see more downsides of using these env files than using the profile yamls as volumes. So far I just briefly discussed this with @Mtze on slack.